### PR TITLE
Update go module, and copyright

### PR DESCRIPTION
**Pull request by c3y1huang**

### CODE REVIEW

**Comment by c3y1huang** [comment]
I cannot add reviewers.
PTAL @longhorn/maintainer

**Review by shuo-wu — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,14 +1,8 @@
-module github.com/keithalucas/longhorn-spdk
+module github.com/longhorn/longhorn-spdk
 
 go 1.13
```
Comment (joshimoo) [inline go.mod:3]: Let's update the go version to v1.17 same as in the manager.
This also changes the way modules are handled a bit.

**Review by joshimoo — COMMENTED** [review]
@innobead @yasker can you update the repositories permissions, doesn't seem like maintainers can approve/request reviewers.

Do we want to keep the copyright notices per file? Seems redundant (since all of it is MIT licensed) and different from the other longhorn repositories.



Let's update the go version to 1.17 to be in sync with the other longhorn repositories, this also changes the module handling a bit.

File: go.mod (hunk 1 above)
Comment (c3y1huang) [reply go.mod]: Updated.

**Comment by c3y1huang** [comment]
> Do we want to keep the copyright notices per file? Seems redundant (since all of it is MIT licensed) and different from the other longhorn repositories.

Removed.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
Thanks @c3y1huang 👍

**Pull request merged**